packages/gatsby-mdx/gatsby-node.js

```diff
@@ -1,120 +1,26 @@
 const path = require("path");
-const fs = require("fs-extra");
-const merge = require("lodash/merge");
-const escapeStringRegexp = require("escape-string-regexp");
 const defaultOptions = require("./utils/default-options");
-const extractExports = require("./utils/extract-exports");
 const mdx = require("./utils/mdx");
 
 /**
  * Create Mdx nodes from MDX files.
  */
-exports.onCreateNode = require("./on-create-node");
+exports.onCreateNode = require("./gatsby/on-create-node");
```

Comment:
there's something funky going on in this PR, does it need a rebase or had a faulty git merge? These changes are already in master so not sure why they're showing up in this PR.